Mage.Tests/src/test/java/org/mage/test/cards/abilities/activated/TokenImplActivatedAbilityTest.java

```diff
@@ -38,7 +38,7 @@
  *
  * @author LevelX2
  */
-public class TokenActivatedAbilityTest extends CardTestPlayerBase {
```

Comment:
override -- that's token (and some other) have old "@Override public xxx copy()" but all new one do not have override for copy. Is that important?